src/commands/server/start.ts

```diff
@@ -43,6 +43,11 @@ export default class Start extends Command {
       default: Start.getTemplatesDir(),
       env: 'CHE_TEMPLATES_FOLDER'
     }),
+    resources: string({
+      description: 'Path to the resources folder',
```

Comment:
What are resources and templates folders for chectl... It's not clear for user how to use them I believe
Che Templates is out of scope the current PR but since you introduce resources folder, I would like to think more about UX here.

Reply:
Yes, that's not clear, but it is the same for files under templates folder. I can move cert-manager yaml files into templates folder, but I think that for CA certificate generation job it should be another location.

Reply:
@mmorhun 
What do you want to store under `resources` folder?

Reply:
@tolusha I've moved content of `resources` folder into `templates` folder.